includes/class-edd-customer.php

```diff
@@ -1082,29 +1082,27 @@ public function get_emails() {
 	 *
 	 * @since 3.0
 	 *
-	 * @param string $type Address type. Default primary.
+	 * @param boolean $is_primary Whether the address is the primary address. Default true.
 	 *
 	 * @return array|\EDD\Customers\Customer_Address Object if primary address requested, array otherwise.
```

Comment:
```suggestion
	 * @return array|\EDD\Customers\Customer_Address|null Object if primary address requested, array otherwise. Null if no result for primary address.
```